apps/web/testing/mocks/auth.js

```diff
@@ -0,0 +1,27 @@
+import { installSessionMock } from './session.js';
+
+/**
+ * Install an authenticated user account to the session with membership to the
```

Comment:
Pretty please add more docs says that the object structure (idTokenClaims and all that) is coming from Azure MSAL and not set by us. It helps putting into context the structure and naming.

Reply:
Ok – I think we can describe this by introducing an extended jsdoc type definition for azure's `AccountInfo` type, tailored to the PINS response.